checkov/common/bridgecrew/vulnerability_scanning/integrations/twistcli.py

```diff
@@ -50,12 +51,18 @@ def report_results(self, twistcli_scan_result: Dict[str, Any], file_path: Path,
             get_default_post_headers(bc_integration.bc_source, bc_integration.bc_source_version),
             {"Authorization": self.get_bc_api_key()},
         )
-        response = requests.request("POST", f"{self.vulnerabilities_base_url}/report", headers=headers, json=payload)
-        try:
-            response.raise_for_status()
-        except requests.exceptions.HTTPError as ex:
-            logging.error(f"HTTP error on request {self.vulnerabilities_base_url}/report, payload:\n{payload}")
-            raise ex
+
+        for i in range(2):
+            response = requests.request("POST", f"{self.vulnerabilities_base_url}/report", headers=headers, json=payload)
+            try:
+                response.raise_for_status()
+                break
+            except requests.exceptions.HTTPError as ex:
+                logging.error(f"HTTP error on request {self.vulnerabilities_base_url}/report, payload:\n{payload}")
+                if ex.response.status_code >= 500 and i != 1:
+                    time.sleep(2000)
```

Comment:
isn't it 2000 seconds? did you want to wait 2 seconds or 20?

Reply:
my bad 😅